build-system/release-workflows/cut-nightly.js

```diff
@@ -14,6 +14,17 @@ const params = {owner: 'ampproject', repo: 'amphtml'};
 // Permanent external ID as assigned by the GitHub Actions runner.
 const GITHUB_EXTERNAL_ID = 'be30aa50-41df-5bf3-2e88-b5215679ea95';
 
+const CHECKS_TO_CHECK = [
```

Comment:
Disagree because we could add a check that has nothing to do with nightly and then we'd have to remember to add it here. It's a small detail really...

Reply:
I could argue the exact opposite - we could be adding a check that should be a release blocker and forget to add it here. The default here should be to fail a release cut fast if there's any doubt